charts/sysdig/values.yaml

```diff
@@ -332,6 +332,8 @@ nodeAnalyzer:
     # The list of directories to inspect during the scan, expressed as a comma separated list.
     # dirsToScan: "/etc,/var/lib/dpkg,/usr/local,/usr/lib/sysimage/rpm,/var/lib/rpm,/lib/apk/db"
 
+    hostBase: "/host"
```

Comment:
Are we sure we need to expose `hostBase` as an option in values? I see the mountPath for root-vol is hardcoded to /host in https://github.com/luca-sd/charts/blob/45e7278014cd734dedb4c7e8b4cb6a493a734d81/charts/sysdig/templates/daemonset-node-analyzer.yaml#L214, so wouldn't it make sense to just hardcode this value in the configmap?

Reply:
It makes sense but I've got a question: can you see issues in case someone used a configmap that has this environment variable updating with a daemonset that doesn't have it exposed anymore?